example/android/src/main/java/com/splendo/kaluga/example/permissions/storage/StoragePermissionsDemoActivity.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright (c) 2020. Splendo Consulting B.V. The Netherlands
+ 
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+ 
+      http://www.apache.org/licenses/LICENSE-2.0
+ 
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ 
+ */
+
+package com.splendo.kaluga.example.permissions.storage
+
+import com.splendo.kaluga.example.permissions.PermissionsDemoActivity
+import com.splendo.kaluga.permissions.Permission
+
+class StoragePermissionsDemoActivity : PermissionsDemoActivity<Permission.Storage>() {
+
+    override val permission = Permission.Storage(true)
```

Comment:
Not clear what is `true` flat does

Reply:
added explicit argument flags for clarity. Also added default constructors to reduce need to set these flags when they are not used